src/scripts/main.js

```diff
@@ -1,3 +1,52 @@
-'use strict';
+import { createForm } from './createForm';
+import { sortTableBody } from './tableActions';
 
-// write code here
+const sort = sortTableBody();
```

Comment:
The `sortTableBody` function is being called immediately and its return value is assigned to `sort`. Ensure that `sortTableBody` returns a function that can be used for sorting, otherwise this might lead to unexpected behavior.